packages/node_modules/@webex/plugin-meetings/src/media/index.js

```diff
@@ -61,14 +61,37 @@ Media.generateLocalMedias = (mediaId, audio, video) => {
 };
 
 Media.getLocalMedia = (options) => {
-  const {sendAudio, sendVideo} = options;
+  const {sendAudio, sendVideo, sendShare} = options;
   // we return undefined here as the second param until screen share is implemented
-  return Promise.resolve().then(() => Media.getMedia(sendAudio, sendVideo).then((media) => [media, undefined]));
-};
 
-Media.addMedia = (mediaDirection, pcMgr, audioVideo, screen) => {
+  if (sendAudio || sendVideo) {
+    return Media.getMedia(sendAudio, sendVideo);
+  }
+
+  if (sendShare) {
```

Comment:
Is it not possible to do both audio/video and share at the same time? Or is that just handled elsewhere?